.github/workflow/release.yml

```diff
@@ -0,0 +1,141 @@
+name: 𝌚 Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      force-publish:
+        required: true
+        type: boolean
+        description: Publish Releases at Anytime
+
+  workflow_run:
+    workflows: [ 🧪 Tests and Checks ]
+    branches: [main]
+    types: [ completed ]
+
+concurrency:
+  group: {{ "${{ github.workflow " }}}}-{{ "${{ github.ref " }}}}
+  cancel-in-progress: true
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    if: >
+      github.ref == 'refs/heads/main' &&
+      github.repository_owner == '{{github-name}}' &&
+      github.event.workflow_run.conclusion == 'success'
+    outputs:
+      release_created: {{ "${{ steps.release.outputs.release_created " }}}}{% if auditable %}
+      tag: {{ "${{ steps.release.outputs.tag_name " }}}}{% endif %}
+
+    steps:
+      - name: Run release-please
+        id: release
+        uses: google-github-actions/release-please-action@v3
+        with:
+          token: {{ "${{ secrets.GITHUB_TOKEN " }}}}
+          default-branch: main
+          command: manifest
+          extra-files: |
+            README.md
+{% if auditable %}
```

Comment:
these are copied from the template for generation and won't work in this context.